### PR TITLE
Commenting. Add comment use case implementation.

### PR DESCRIPTION
**Pull request by AndreiGanichev**
Implement AddMeetingComment use case
Add PriceListItem permissions to SeedDatatbase script

### CODE REVIEW

File: src/Database/CompanyName.MyMeetings.Database/Scripts/CreateStructure.sql [hunk 1]
```diff
@@ -347,6 +347,22 @@ CREATE TABLE [meetings].[MeetingAttendees] (
...
+
+GO
+CREATE TABLE meetings.MeetingComments
+(
+	[Id] UNIQUEIDENTIFIER NOT NULL,
+    [MeetingId] UNIQUEIDENTIFIER NOT NULL,
+    [AuthorId] UNIQUEIDENTIFIER NOT NULL,
+    [InReplyToCommentId] UNIQUEIDENTIFIER NULL,
+    [Comment] VARCHAR(300) NULL,
+    [CreateDate] DATETIME NOT NULL,
```
Comment (kgrzybek) [inline src/Database/CompanyName.MyMeetings.Database/Scripts/CreateStructure.sql:362]: Please fix indents

File: src/Database/CompanyName.MyMeetings.Database/meetings/Tables/MeetingComments.sql [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿CREATE TABLE meetings.MeetingComments
+(
+	[Id] UNIQUEIDENTIFIER NOT NULL,
+    [MeetingId] UNIQUEIDENTIFIER NOT NULL,
+    [AuthorId] UNIQUEIDENTIFIER NOT NULL,
+    [InReplyToCommentId] UNIQUEIDENTIFIER NULL,
+    [Comment] VARCHAR(300) NULL,
+    [CreateDate] DATETIME NOT NULL,
```
Comment (kgrzybek) [inline src/Database/CompanyName.MyMeetings.Database/meetings/Tables/MeetingComments.sql:8]: Please fix indents

File: src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        private readonly string _comment;
+
+        public CommentTextMustBeProvidedRule(string comment)
+        {
+            _comment = comment;
+        }
+
+        public bool IsBroken() => _comment == null;
+
+        public string Message => "Comment text must br provided.";
```
Comment (kgrzybek) [inline src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs:16]: typo

File: src/Modules/Meetings/Domain/MeetingComments/MeetingComment.cs [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+            
+            this.AddDomainEvent(new MeetingCommentCreatedDomainEvent(Id));
+        }
+        
+        private MeetingComment()
+        {
+            // Only for EF.
+        }
+
+        public static MeetingComment Create(MeetingId meetingId, MemberId authorId, string comment)
```
Comment (kgrzybek) [inline src/Modules/Meetings/Domain/MeetingComments/MeetingComment.cs:50]: Can be internal

File: src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+    public class CommentTextMustBeProvidedRule : IBusinessRule
+    {
+        private readonly string _comment;
+
+        public CommentTextMustBeProvidedRule(string comment)
+        {
+            _comment = comment;
+        }
+
+        public bool IsBroken() => _comment == null;
```
Comment (kgrzybek) [inline src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs:14]: Maybe `string.IsNullOrEmpty(_comment)` to check against empty string too?

File: src/Modules/Meetings/Tests/UnitTests/MeetingComments/MeetingCommentTests.cs [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+namespace CompanyName.MyMeetings.Modules.Meetings.Domain.UnitTests.MeetingComments
+{
+    [TestFixture]
+    public class MeetingCommentTests : TestBase
+    {
+        [Test]
+        public void CreateMeetingComment_IsSuccessful()
+        {
+            var meetingComment = MeetingComment.Create(
```
Comment (kgrzybek) [inline src/Modules/Meetings/Tests/UnitTests/MeetingComments/MeetingCommentTests.cs:18]: To change after marking Create as internal.

File: src/Database/CompanyName.MyMeetings.Database/Scripts/CreateStructure.sql (hunk 1 above)
Comment (AndreiGanichev) [reply src/Database/CompanyName.MyMeetings.Database/Scripts/CreateStructure.sql]: fixed

File: src/Database/CompanyName.MyMeetings.Database/meetings/Tables/MeetingComments.sql (hunk 2 above)
Comment (AndreiGanichev) [reply src/Database/CompanyName.MyMeetings.Database/meetings/Tables/MeetingComments.sql]: fixed

File: src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs (hunk 3 above)
Comment (AndreiGanichev) [reply src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs]: fixed

File: src/Modules/Meetings/Tests/UnitTests/MeetingComments/MeetingCommentTests.cs (hunk 6 above)
Comment (AndreiGanichev) [reply src/Modules/Meetings/Tests/UnitTests/MeetingComments/MeetingCommentTests.cs]: move tests for add comment logic to meeting tests

File: src/Modules/Meetings/Domain/MeetingComments/MeetingComment.cs (hunk 4 above)
Comment (AndreiGanichev) [reply src/Modules/Meetings/Domain/MeetingComments/MeetingComment.cs]: done

File: src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs (hunk 5 above)
Comment (AndreiGanichev) [reply src/Modules/Meetings/Domain/MeetingComments/Rules/CommentTextMustBeProvidedRule.cs]: i agree, thanks

**Pull request merged**